src/site/antora/modules/ROOT/examples/hibernate/log4j2.properties

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+appender.0.type = Console
+appender.0.name = CONSOLE
+appender.0.layout.type = PatternLayout
+
+# Log just the SQL
+logger.0.name = org.hibernate.SQL
+logger.0.level = DEBUG
+
+# Log JDBC bind parameters and extracted values
+logger.1.name = org.hibernate.type
+logger.1.level = TRACE
+logger.2.name = org.hibernate.orm.jdbc.bind
+logger.2.level = TRACE
+logger.3.name = org.hibernate.orm.jdbc.bind
+logger.3.level = TRACE
```

Comment:
I love that you also provided a `*.properties` file! :100:

Reply:
Friends don't let friends suffer alone. :sunglasses: